decidim/decidim
# Add a privacy warning on non-transparent private spaces

#### :tophat: What? Why?

As an administrator, while I'm on a participatory space that's private and non-transparent, I could expect that the attached files that I upload are somehow protected, as it's a private space. 

This is not the case, as any participant could share the URL of the file or even download it and share it by other means. 

This PR adds a callout warning about this potential security problem.


#### Testing

1. Sign in as admin
2. Edit an assembly with 
  - Private space: check
  -Is transparent: uncheck
3. Go to attachments files

### :camera: Screenshots

![image](https://user-images.githubusercontent.com/717367/151176986-a0d5d58b-cc01-46ed-a032-c2ff94808443.png)

:hearts: Thank you!
